test/index-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Oh yeah, this is why I can't find it in my code. It is deprecated rule.

http://eslint.org/docs/rules/global-strict.html